### ISSUE TITLE
Unable to fetch resources from the web when setting custom branding 

### ISSUE THREAD

**Issue by vmm-afonso** [opensearch-project/OpenSearch-Dashboards#3433]
I've been trying to customize the branding of opensearch-dashboards and I'm editing the opensearch_dashboards.yml to grab a few .svgs from the the internet without much success. 

-----------------------------MODS TO THE FILE EG-----------------------------
opensearchDashboards.branding:
  mark:
    defaultUrl: "https://cdn.worldvectorlogo.com/logos/elastic-kibana.svg"

--------------------------------ERROR MESSAGE--------------------------------
"tags":["error","branding"],"pid":1,"message":"mark default URL was not found or invalid. Using default branding."}

-----------------------------------------------------------------------------------

Is there anyway for me to fetch these images from the web or does it only work for locally hosted?

**Comment by designermonkey** [comment]
Does your config value get rewritten to be http? Mine does, and is causing invalid uris.

**Comment by vmm-afonso** [comment]
Ohh I forgot to close this issue. It's solved now. I created a secretMount with a folder containing my svgs and the path /usr/share/opensearch-dashboards/assets , after that I simply edited my opensearch_dashboards.yml to change the DefaultURLs to "https://localhost:5601/ui/assets/example.svg". In my first attempts I wasn't able to see the logos but that was bcs my dumbass had created a route for opensearch-dashboards but was still specifying "localhost:5601" instead of the actual route in the URLs
